automation.md

```diff
@@ -0,0 +1,25 @@
+# Automation
+
+## Activated automations:
+
+ - **Out of the box github automerge** will merge PR once everything is green. It is activable from a pull request page.
```

Comment:
A small edit:
**Out of the box, github automerge** will merge a PR once everything is green. It is activated from a pull request page.